SPECS/filesystem/filesystem.spec

```diff
@@ -1,715 +1,983 @@
+
+* Fri Jul 21 2000 Bill Nottingham <notting@redhat.com>
+- add /usr/share/empty
+
+* Wed Jul 12 2000 Prospector <bugzilla@redhat.com>
+- automatic rebuild
+
+* Wed Jun 28 2000 Preston Brown <pbrown@redhat.com>
+- remove /usr/doc
+
+* Thu Jun 22 2000 Preston Brown <pbrown@redhat.com>
+- remove /usr/info
+
+* Sun Jun 19 2000 Bill Nottingham <notting@redhat.com>
+- remove /usr/man
+
+* Sat Jun 17 2000 Bill Nottingham <notting@redhat.com>
+- /var/spool/lpd should have normal perms (#12272)
+
+* Tue Jun  6 2000 Bill Nottingham <notting@redhat.com>
+- add /etc/skel
+
+* Thu Jun 01 2000 Preston Brown <pbrown@redhat.com>
+- add /var/spool/lpd to filesystem, owned by user/group lp, tight permissions
+
+* Tue May 23 2000 Trond Eivind Glomsrød <teg@redhat.com>
+- Added /etc/xinetd.d
+
+* Mon May 15 2000 Preston Brown <pbrown@redhat.com>
+- /etc/opt, /usr/share/{info,man/man*,misc,doc} (FHS 2.1)
+- added /var/games.  Data should move from /var/lib/games to there (FHS 2.1)
+- bump version up to 2.0 already!
+
+* Thu Apr 13 2000 Jakub Jelinek <jakub@redhat.com>
+- removed /var/state, added /var/opt, /var/mail for FHS 2.1 compliance
+
+* Mon Aug 28 1999 Preston Brown <pbrown@redhat.com>
+- added /opt, /var/state, /var/cache for FHS compliance (#3966)
+
+* Sun Mar 21 1999 Cristian Gafton <gafton@redhat.com> 
+- auto rebuild in the new build environment (release 4)
+
+* Sun Jan 17 1999 Jeff Johnson <jbj@redhat.com>
+- don't carry X11R6.1 as directory on sparc.
+- /var/tmp/build root (#811)
+
+* Wed Jan 13 1999 Preston Brown <pbrown@redhat.com>
+- font directory didn't belong, which I previously misunderstood.  removed.
+
+* Fri Nov 13 1998 Preston Brown <pbrown@redhat.com>
+- /usr/share/fonts/default added.
+
+* Fri Oct  9 1998 Bill Nottingham <notting@redhat.com>
+- put /mnt/cdrom back in
+
+* Wed Oct  7 1998 Bill Nottingham <notting@redhat.com>
+- Changed /root to 0750
+
+* Wed Aug 05 1998 Erik Troan <ewt@redhat.com>
+- added /var/db
+- set attributes in the spec file; don't depend on the ones in the cpio
+  archive
+- use a tarball instead of a cpioball
+
+* Mon Apr 27 1998 Prospector System <bugs@redhat.com>
+- translations modified for de, fr, tr
+
+* Tue Sep 09 1997 Erik Troan <ewt@redhat.com>
+- made a noarch package
+
+* Wed Jul 09 1997 Erik Troan <ewt@redhat.com>
+- added /
+
+* Wed Apr 16 1997 Erik Troan <ewt@redhat.com>
+- Changed /proc to 555
+- Removed /var/spool/mqueue (which is owned by sendmail)
+
+* Mon Mar 04 2024 Dan Streetman <ddstreet@microsoft.com> - 1.1-20
+- move filesystem-asc stuff into asc
+- move /etc/modprobe.d into kmod package
+- move /etc/mtab from filesystem to util-linux package
+- move /var/log/* files from filesystem to systemd package
+- remove opensuse-style 'proxy' config file
+- remove unused /etc/sysconfig/clock file
+- remove unused opensuse-style /etc/sysconfig/console file
+- move some files into setup package
 
-%changelog
 * Wed Feb 28 2024 Dan Streetman <ddstreet@microsoft.com> - 1.1-19
```

Comment:
Merge leftover? I think `rpmbuild` will complain about out-of-order changelog entries.

Reply:
i'd intentionally merged the changelogs, but i can drop all the old entries, i don't care...

Reply:
Not sure what you meant but I was referring to the changelog entries being out of order. I see they are ordered now, so we should be fine.